server/src/main/java/org/apache/iotdb/db/query/control/FileReaderManager.java

```diff
@@ -246,7 +246,7 @@ public synchronized boolean contains(TsFileResource tsFile, boolean isClosed) {
   public synchronized void writeFileReferenceInfo() {
     DEBUG_LOGGER.info("[closedReferenceMap]\n");
     for (Map.Entry<String, AtomicInteger> entry : closedReferenceMap.entrySet()) {
-      DEBUG_LOGGER.info(String.format("\t%s: %d\n", entry.getKey(), entry.getValue().get()));
+      DEBUG_LOGGER.info(String.format("\t%s: %d%n", entry.getKey(), entry.getValue().get()));
```

Comment:
Why change `\n` to `%n` here?

Reply:
Please check https://rules.sonarsource.com/java/RSPEC-3457 and https://stackoverflow.com/questions/31981136/difference-between-n-and-n-for-printing-a-new-line-in-java

Reply:
I just learned that %n should be used in place of \n to produce the platform-specific line separator

Reply:
Get it~